v1.1.44 changelog updates

dtardoin opened the pull request:
## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

Commit:
1.1.44 changelog updates

runleonarun committed:
Update dbt-cloud-changelog.md

runleonarun committed:
Update dbt-cloud-changelog.md

runleonarun commented on website/docs/docs/dbt-cloud/dbt-cloud-changelog.md:
```diff
@@ -4,6 +4,16 @@ id: "cloud-changelog"
 ---
+## dbt Cloud v1.1.44 (February 2nd, 2022)
+
+Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands. 
```
```suggestion
Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands.  

Updates to the scheduler include localized timestamps and how long a run remains blocked, which pinpoints how long a job is unable to run. For example, a run might be blocked waiting for a previous job to finish. Look for these changes in the Timing tab for run details. 
```

runleonarun committed:
Update website/docs/docs/dbt-cloud/dbt-cloud-changelog.md

runleonarun commented on website/docs/docs/dbt-cloud/dbt-cloud-changelog.md:
```diff
@@ -4,6 +4,18 @@ id: "cloud-changelog"
+
+* The Command Bar recommendation actions now support dbt Core v1.0 commands. For example, you now see "build" and the "--select" argument.
+* Created and Finished times for runs now use localized timestamps.
+* The scheduler now shows blocked time for a run, which pinpoints the time a job is unable to run, such as waiting for a previous job to finish.  
```
```suggestion
* The scheduler now shows blocked time for a run.  
```

runleonarun committed:
Update website/docs/docs/dbt-cloud/dbt-cloud-changelog.md

runleonarun commented on website/docs/docs/dbt-cloud/dbt-cloud-changelog.md:
```diff
@@ -4,6 +4,18 @@ id: "cloud-changelog"
+
+#### Performance improvements and enhancements
+
+* The Command Bar recommendation actions now support dbt Core v1.0 commands. For example, you now see "build" and the "--select" argument.
```
```suggestion
* The Command Bar recommendation actions now support dbt Core v1.0 commands. 
```

runleonarun commented on website/docs/docs/dbt-cloud/dbt-cloud-changelog.md:
```diff
@@ -4,6 +4,18 @@ id: "cloud-changelog"
 ---
+## dbt Cloud v1.1.44 (February 2nd, 2022)
+
+Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands.  
```
```suggestion
Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands, for example, you now see "build" and the "--select" argument.
```

runleonarun committed:
Apply suggestions from code review

runleonarun commented on website/docs/docs/dbt-cloud/dbt-cloud-changelog.md:
```diff
@@ -4,6 +4,18 @@ id: "cloud-changelog"
 ---
+## dbt Cloud v1.1.44 (February 2nd, 2022)
+
+Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands, for example, you now see "build" and the "--select" argument.
```
```suggestion
Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands, which means you will now see recommendations for "build" and the "--select" argument.
```

runleonarun committed:
Update website/docs/docs/dbt-cloud/dbt-cloud-changelog.md

runleonarun commented on website/docs/docs/dbt-cloud/dbt-cloud-changelog.md:
```diff
@@ -4,6 +4,18 @@ id: "cloud-changelog"
+
+#### Performance improvements and enhancements
+
+* The Command Bar recommendation actions now support dbt Core v1.0 commands. 
```
```suggestion
* The Command bar recommendation actions now support dbt Core v1.0 commands. 
```

runleonarun committed:
Apply suggestions from code review

schottj committed:
Update dbt-cloud-changelog.md

made changes as this release won't have the new timestamps for the run details after all.

runleonarun reviewed (approved):
LGTM
